### ISSUE TITLE
Crash on Android running SDK 18 on initializing Rollbar 1.2.1

### ISSUE THREAD

**Issue by TheIronMarx** [rollbar/rollbar-java#161]
Initializing Rollbar via `Rollbar.init(context, client_key, environment);`, my application crashes on Android SDK 18 devices due to an apparent attempt to access `java.util.Objects`, which isn't introduced until Android SDK 19. Below is the stacktrace.

```
E/AndroidRuntime: FATAL EXCEPTION: main
    java.lang.NoClassDefFoundError: java.util.Objects
        at com.rollbar.notifier.Rollbar.handleUncaughtErrors(Rollbar.java:91)
        at com.rollbar.notifier.Rollbar.handleUncaughtErrors(Rollbar.java:82)
        at com.rollbar.notifier.Rollbar.<init>(Rollbar.java:54)
        at com.rollbar.notifier.Rollbar.<init>(Rollbar.java:46)
        at com.rollbar.android.Rollbar.<init>(Rollbar.java:271)
        at com.rollbar.android.Rollbar.<init>(Rollbar.java:201)
        at com.rollbar.android.Rollbar.init(Rollbar.java:112)
        at com.rollbar.android.Rollbar.init(Rollbar.java:94)
        at com.rollbar.android.Rollbar.init(Rollbar.java:80)
        at com.rollbar.android.Rollbar.init(Rollbar.java:67)
        at com.mycompany.myapp.CustomApplication.initRollbar(CustomApplication.java:107)
        at com.mycompany.myapp.CustomApplication.onCreate(CustomApplication.java:82)
        at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1007)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4444)
        at android.app.ActivityThread.access$1300(ActivityThread.java:141)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1316)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:137)
        at android.app.ActivityThread.main(ActivityThread.java:5103)
        at java.lang.reflect.Method.invokeNative(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:525)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:737)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
        at dalvik.system.NativeStart.main(Native Method)
```

**Comment by basoko** [comment]
Hi @TheIronMarx ,

Thanks for reporting this, and sorry for facing this issue. We have removed all the uses of java.util.Objects, and published a new version 1.3.1 that should fix it.

It should be available on Maven soon, so let us know if this release fixes your issue.

Thanks!

**Comment by TheIronMarx** [comment]
Cheers!
